app/controllers/admin/rooms_controller.rb

```diff
@@ -14,12 +15,13 @@ def new
 
     def create
       @room = @venue.rooms.new(room_params)
-      if @room.save
-        redirect_to admin_conference_venue_rooms_path(conference_id: @conference.short_title),
-                    notice: 'Room successfully created.'
-      else
-        flash[:error] = "Creating Room failed: #{@room.errors.full_messages.join('. ')}."
-        render :new
+      respond_to do |format|
+        if @room.save
+          flash[:notice] = 'Room successfully created.'
+        else
+          flash[:error] = "Creating Room failed: #{@room.errors.full_messages.join('. ')}."
+        end
+        format.js{ render 'message' }
```

Comment:
Previously, I used it to render flash message without unobtrusive flash. I will remove message partial and explicit render from controller. :+1: